test/frontend/test_node.py

```diff
@@ -94,24 +116,44 @@ def do_test_expected(self):
 ("test_or", tf.logical_or, "LogicalOr", [get_rnd([10, 10], dtype=np.bool_), get_rnd([10, 10], dtype=np.bool_)], {}),
 ("test_pow", tf.pow, "Pow", [get_rnd([10, 10]), get_rnd([10, 10])], {}),
 ("test_pad", tf.pad, "Pad", [get_rnd([2, 3]), [[1, 1,], [2, 2]]], {"mode": "constant"}),
+("test_pad", tf.pad, "Pad", [get_rnd([10, 2, 2, 3]), [[0,0], [2, 2], [2, 2], [0,0]]], {"mode": "constant"}),
```

Comment:
Test name has to be unique. Otherwise only one of these gets run.

Reply:
Oh, thanks to notice it to me.
Do you want to make to change before to merge ? 
Or do you want I make the modification ?

Reply:
Could you modify them to have unique names?

Reply:
I can do it ASAP.